lib/nudge_web/controllers/site_controller.ex

```diff
@@ -0,0 +1,28 @@
+defmodule NudgeWeb.SiteController do
+  use NudgeWeb, :controller
+
+  def index(conn, _params) do
+    sites = Nudge.Accounts.list_sites()
+    render(conn, "index.html", sites: sites)
+  end
+
+  def new(conn, _params) do
+    changeset = Nudge.Accounts.change_site(%Nudge.Accounts.Site{})
+    render(conn, "new.html", changeset: changeset)
+  end
+  def create(conn, %{"site" => site_params}) do
+    case Nudge.Accounts.create_site(site_params) do
```

Comment:
When we create a site, it needs to associated to the user that is logged in.
Every site should belong to a user.

You will get the logged user from `conn.assigns.current_user`.

For details, see `lib/nudge_web/plugs/get_current_user.ex`